ambariclient/models.py

```diff
@@ -348,7 +348,7 @@ def stop(self):
 
     def restart(self):
         """Restarts this component on its host, if already installed and started."""
-        if self.state in ('STARTED', 'STOPPED', 'UNKNOWN'):
+        if self.state in ('STARTED', 'STOPPED', 'UNKNOWN', 'INSTALLED'):
```

Comment:
I think that about covers all the possible states now, so maybe we can just remove the check entirely and let the API tell them if they did something dumb?